Formal integral tests needed for interpolate_shells: > 0

As demonstrated through the bug from issue #1463, we need tests of the formal integral that also cover the case where "interpolate_shells > 0".

@chvogl While updating the refence data, the integral test were failing due to the following error - 
```
            simulation.runner.hdf_properties = [
                "j_blue_estimator",
                "spectrum",
                "spectrum_integrated",
            ]
>           simulation.runner.to_hdf(tardis_ref_data, "", self.name)

tardis/tests/test_tardis_full_formal_integral.py:67: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
tardis/io/util.py:319: in to_hdf
    self.to_hdf_util(file_path_or_buf, buff_path, data, overwrite)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

path_or_buf = <class 'pandas.io.pytables.HDFStore'>
File path: /media/arib/D/tardis/tardis-refdata/unit_test_data.h5

path = 'test_runner_simple_integral_macroatom'
elements = {'j_blue_estimator': array([[0., 0., 0., ..., 0., 0., 0.],
       [0., 0., 0., ..., 0., 0., 0.],
       [0., 0., 0., ....object at 0x7f9549eb7490>, 'spectrum_integrated': <tardis.montecarlo.spectrum.TARDISSpectrum object at 0x7f95575bc220>}
overwrite = False, complevel = 9, complib = 'blosc'

    @staticmethod
    def to_hdf_util(
        path_or_buf, path, elements, overwrite, complevel=9, complib="blosc"
    ):
        """
        A function to uniformly store TARDIS data to an HDF file.
    
        Scalars will be stored in a Series under path/scalars
        1D arrays will be stored under path/property_name as distinct Series
        2D arrays will be stored under path/property_name as distinct DataFrames
    
        Units will be stored as their CGS value
    
        Parameters
        ----------
        path_or_buf : str or pandas.io.pytables.HDFStore
            Path or buffer to the HDF file
        path : str
            Path inside the HDF file to store the `elements`
        elements : dict
            A dict of property names and their values to be
            stored.
        overwrite : bool
            If the HDF file path already exists, whether to overwrite it or not
    
        Notes
        -----
        `overwrite` option doesn't have any effect when `path_or_buf` is an
        HDFStore because the user decides on the mode in which they have
        opened the HDFStore ('r', 'w' or 'a').
        """
        try:  # when path_or_buf is a str, the HDFStore should get created
            buf = pd.HDFStore(path_or_buf, complevel=complevel, complib=complib)
        except TypeError as e:
            if e.message == "Expected bytes, got HDFStore":
                # when path_or_buf is an HDFStore buffer instead
                buf = path_or_buf
            else:
                raise e
        else:  # path_or_buf was a str
            if os.path.exists(path_or_buf) and not overwrite:
                buf.close()
>               raise FileExistsError(
                    "The specified HDF file already exists. If you still want "
                    "to overwrite it, set option overwrite=True"
                )
E               FileExistsError: The specified HDF file already exists. If you still want to overwrite it, set option overwrite=True
```
Any reason why this must be happening?

--- a/tardis/tests/test_tardis_full_formal_integral.py
+++ b/tardis/tests/test_tardis_full_formal_integral.py
@@ -64,7 +64,9 @@ class TestRunnerSimpleFormalInegral:
                 "spectrum",
                 "spectrum_integrated",
             ]
-            simulation.runner.to_hdf(tardis_ref_data, "", self.name)
+            simulation.runner.to_hdf(
+                tardis_ref_data, "", self.name, overwrite=True
+            )
             pytest.skip("Reference data was generated during this run.")
 
     @pytest.fixture(scope="class")

--- a/tardis/tests/test_tardis_full_formal_integral.py
+++ b/tardis/tests/test_tardis_full_formal_integral.py
@@ -7,8 +7,6 @@ from astropy.tests.helper import assert_quantity_allclose
 from tardis.simulation.base import Simulation
 from tardis.io.config_reader import Configuration
 
-pytestmark = pytest.mark.skip(reason="memory problem")
-
 config_line_modes = ["downbranch", "macroatom"]
 interpolate_shells = [-1, 30]
 

